index.html

```diff
@@ -5497,6 +5506,44 @@ <h2>Base Context</h2>
 }
 </pre>
 
+      </section>
+      <section class="informative appendix">
+        <h2>Differences between Contexts and CredentialSchemas</h2>
+        <p>
+The primary purpose of a <code>credentialSchema</code> is different from the
+primary purpose of <code>@context</code>. A <code>credentialSchema</code> is
+useful for defining the data shape and type of a <code>Verifiable
+Credential</code>, whereas [[JSON-LD]] and a <code>@context</code> in a
+verifiable credential are best used only for conveying the semantics and term 
+definitions of the data.
```

Comment:
Replace the above paragraph with
The primary purpose of the <code>credentialSchema</code> property is to define the properties of the <code>VerifiableCredential</code> in which it appears, and the dataypes of each these properties. The use of <code>credentialSchema</code> is optional.